### DISCUSSION TITLE
Request to modify the SensorCard component.

### DISCUSSION THREAD

**Discussion by julian20015464** [shannonhochkins/ha-component-kit#80] [Q&A]
Good morning, I would like to request a favor. I'm working on my thesis project and I've chosen to work with your library to create my web page with Home Assistant. What I'd like to ask is if it's possible to have the "x" and "y" values in the SensorCard component, where "x" represents the sensor value, and "y" could be the time. This is to provide more informative graphs.

**Comment by shannonhochkins** [comment]
There's already a hook to provide this, useHistory from core

**Reply by shannonhochkins** [reply]
if you're using `useEntity` this is also available under `entity.history` if the entity has history data.
